### ISSUE TITLE
Firefox port

### ISSUE THREAD

**Issue by Nabukodonosor** [evanshultz/neater-bookmarks#109]
Would you mind porting this awesome addon to Firefox please?

**Comment by ahallaha** [comment]
I got most everything moved off of Google's apis plus deprecated calls updated but it will still need quite a bit of work. I could really use help from someone with experience in localization. Firefox's add-on linter was flagging the ar and zh-HK localization files as invalid json as well as being unable to read neat.css due to the bidi properties.
https://github.com/ahallaha/neater-bookmarks-firefox

**Comment by Nabukodonosor** [comment]
Can you publish the first version without those problematic languages?

**Comment by pacosalasv** [comment]
Yes — a Firefox-compatible rebuilt version is now available here:

👉 **[Neater Bookmarks Rebuilt 1.0.0 — Firefox + modern browser compatibility rebuild](https://github.com/evanshultz/neater-bookmarks/issues/112)**

The rebuild includes a dedicated Firefox package, plus a Chrome / Edge / Brave package and the source package.

For Firefox testing:

1. Download the Firefox ZIP from the issue.
2. Unzip it.
3. Open `about:debugging#/runtime/this-firefox`.
4. Click **Load Temporary Add-on**.
5. Select the `manifest.json` file.

Note: this is currently provided as a manual/testing build, not an AMO-published release.

**Comment by Nabukodonosor** [comment]
Your Firefox link is for the Chrome addon.

**Comment by pacosalasv** [comment]
> Your Firefox link is for the Chrome addon.

Thanks for the heads-up. The link has been corrected and verified.
